nylas/client/client.py

```diff
@@ -249,8 +250,35 @@ def free_busy(self, emails, start_at, end_at):
         _validate(resp)
         return resp.json()
 
+    def open_hours(self, emails, days, timezone, start, end):
+        if isinstance(emails, six.string_types):
+            emails = [emails]
+        if isinstance(days, int):
+            days = [days]
+        if isinstance(start, datetime):
+            start = "{hour}:{minute}".format(hour=start.hour, minute=start.minute)
+        if isinstance(start, datetime):
+            end = "{hour}:{minute}".format(hour=end.hour, minute=end.minute)
+        return {
+            "emails": emails,
+            "days": days,
+            "timezone": timezone,
+            "start": start,
+            "end": end,
+            "object_type": "open_hours",
+        }
+
     def availability(
-        self, emails, duration, interval, start_at, end_at, free_busy=None
+        self,
+        emails,
+        duration,
+        interval,
+        start_at,
+        end_at,
+        buffer=None,
+        round_robin=None,
+        free_busy=None,
```

Comment:
Missed this on the first go, free_busy must be an array. If you pass it `None` or don't include it in the call to availability then you'll get an error. This should default to `[]` instead of `None`